apps/game_backend/lib/game_backend/users.ex

```diff
@@ -211,7 +211,48 @@ defmodule GameBackend.Users do
       nil
   """
   def get_dungeon_settlement_level(level_number) do
```

Comment:
I'm not sure if the module users is the one for this kind of operations, specially when this isn't related with the user and this belongs to champions. What do you think?